.travis.yml

```diff
@@ -11,14 +11,13 @@ env:
 
 gemfile:
   - Gemfile
-  - gemfiles/rails_4.0.gemfile
   - gemfiles/rails_4.1.gemfile
   - gemfiles/rails_4.2.gemfile
 
 before_install:
   - wget -O vault.zip -q https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_linux_amd64.zip
   - unzip vault.zip
-  - mkdir ~/bin
+  - mkdir -p ~/bin
```

Comment:
One or more of the images used by Travis CI already have the ~bin directory created, so the previous line was causing the build to always fail for me. I added the ```-p``` flag to add a little bit of flexibility with the underlying image.